docs/stats/index.rst

```diff
@@ -77,8 +80,42 @@ with the Li \& Ma formula:
    >>> significance_on_off(n_on=18, n_off=97, alpha=1. / 10, method='lima')
    2.2421704424844875
 
+Confidence Intervals
+--------------------
+
+Assume you measured 6 counts in a Poissonian counting experiment with an
+expected background :math:`b = 3`. Here's how you compute the 90% upper limit
+on the signal strength :math:`\\mu`:
```

Comment:
Isn't this the use case covered by the [fc_find_acceptance_regino_poisson](http://www.mpi-hd.mpg.de/personalhomes/deil/public/gammapy/issue_367/html/api/gammapy.stats.fc_find_acceptance_region_poisson.html) convenience function?
If yes, I'd suggest using that simpler version on this top-level `gammapy.stats` docs page, and moving the (equivalent?)  code that takes several calls to more general `fc*` functions to the FC docs page. Or keep it here in addition, but also show how to call the simpler `fc_find_acceptance_region_poisson` function here, please.